sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -208,3 +208,7 @@ CREATE TABLE `patient_settings` (
 PRIMARY KEY (`setting_patient`, `setting_label`)
 ) ENGINE=InnoDB;
 #EndIf
+
+#IfNotRow2D list_options list_id lists option_id notification_template
```

Comment:
gonna fix above line up in a quick follow up commit :)
```
#IfNotRow2D list_options list_id Document_Template_Categories option_id notification_template
```

Reply:
Hmm @bradymiller sorry and thanks.
@stephenwaite thanks for catching that last one also.
I think I’ll start waiting for reviews. I did test this and seem to work. But wanted to get it in to continue development.
I’m thinking I should retire soon

Reply:
no prob at all! keep flying :)